jetty-core/jetty-client/src/main/java/org/eclipse/jetty/client/HttpClientTransport.java

```diff
@@ -64,24 +65,38 @@ public interface HttpClientTransport extends ClientConnectionFactory, HttpClient
      * @param origin the destination origin
      * @return a new, transport-specific, {@link HttpDestination} object
      */
-    public Destination newDestination(Origin origin);
+    Destination newDestination(Origin origin);
 
     /**
      * Establishes a physical connection to the given {@code address}.
      *
      * @param address the address to connect to
      * @param context the context information to establish the connection
      */
-    public void connect(SocketAddress address, Map<String, Object> context);
+    void connect(SocketAddress address, Map<String, Object> context);
 
     /**
      * @return the factory for ConnectionPool instances
      */
-    public ConnectionPool.Factory getConnectionPoolFactory();
+    ConnectionPool.Factory getConnectionPoolFactory();
 
     /**
      * Set the factory for ConnectionPool instances.
      * @param factory the factory for ConnectionPool instances
      */
-    public void setConnectionPoolFactory(ConnectionPool.Factory factory);
+    void setConnectionPoolFactory(ConnectionPool.Factory factory);
+
+    /**
+     * @return the {@link InvocationType} associated with this {@code HttpClientTransport}.
```

Comment:
The javadoc should explain that the invocation type is going to be used to figure out how to execute the listeners.

Reply:
Done.